README.md

```diff
@@ -259,6 +266,42 @@ A JSON structured response within the body containing the listing, or an iRODS e
 }
 ```
 
+### /logicalpath
+Deletes a data object or a collection.
+
+**Method** DELETE
+
+**Parameters**
+- logical-path: The absolute path leading to the data object or collection to be deleted.
+- no-trash: Don't send to trash, delete permanently. Optional, defaults to false.
+- recursive: Recursively delete contents of a collection. Optional, defaults to false.
+- unregister: Unregister data objects instead of deleting them. Optional, defaults to false.
+
+**Example CURL command**
+```
+curl -X DELETE -H "Authorization: ${TOKEN}" "http://localhost:80/irods-rest/0.9.1/logicalpath?logical-path=/tempZone/home/rods/hello.cpp&no-trash=1"
+```
+
+**Returns**
+Nothing on success.
+
```

Comment:
Did the example of what is returned on failure get lost somewhere? There's a resolved comment or two that seem to suggest that these existed at some point.

Reply:
Sorry, I'm sort of hopping around tasks. I think Kory was suggesting that I show maybe one example at the top of the document to demonstrate the general structure of the error messages instead of giving an example for every one (since they're all going to come from `make_error_response`) anyway.

Reply:
Ah, I didn't see that. Carry on.